ambry-api/src/main/java/com/github/ambry/network/RequestInfo.java

```diff
@@ -31,6 +32,8 @@ public class RequestInfo {
   private long requestEnqueueTime = -1;
   private long requestSendTime = -1;
   private long responseHeaderReceiveTime = -1;
+  private final AtomicLong networkTimeOutMs = new AtomicLong(0);
```

Comment:
Why atomic? For one `RequestInfo` object, would multiple threads try to update the timeout value?

Reply:
Hmm, yeah was just thinking if it is possible that read and increment could be called in parallel. But, since each `requestInfo` will be accessed by only one request handler thread during `sendAndPoll` operation, may be we don't need Atomic operation. Will remove it.